test/jdk/com/sun/crypto/provider/Cipher/ChaCha20/unittest/Poly1305UnitTestDriver.java

```diff
@@ -49,14 +49,14 @@
  * @key randomness
  * @modules java.base/com.sun.crypto.provider
  * @summary Unit test for IntrinsicCandidate in com.sun.crypto.provider.Poly1305.
- * @run main/othervm -Xcomp -XX:-TieredCompilation java.base/com.sun.crypto.provider.Poly1305IntrinsicFuzzTest
+ * @run main/othervm -Xcomp -XX:-TieredCompilation -XX:+ForceUnreachable java.base/com.sun.crypto.provider.Poly1305IntrinsicFuzzTest
```

Comment:
Don't you need `-XX:+UnlockDiagnosticVMOptions` here as well?

Reply:
Darn. Yes.. one min, will push.. Sorry about the noise. (thought that the fact that it failed on linux meant it wasnt a diagnostic, but it is..)

```
  product(bool, ForceUnreachable, false, DIAGNOSTIC,                        \
          "Make all non code cache addresses to be unreachable by "         \
          "forcing use of 64bit literal fixups")
```